backend/products/models.py

```diff
@@ -13,25 +11,19 @@ class Product(models.Model):
                                    blank=True)
     category = models.CharField(max_length=100, 
                                 choices=category_choices)
-    primary_variant = models.OneToOneField(
-        'ProductVariation', 
-        on_delete=models.CASCADE, 
-        default=None, 
-        null=True,
-        blank=True,)
     
-    # Need to add Primary Variant and seller Id
+    color = models.CharField(max_length=100, blank = True)
+    size = models.CharField(max_length=100, blank = True)
+    price = models.IntegerField(default=0)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return self.name
+        return f'{self.name} {self.color} {self.size}'
     
 
 class ProductVariation(models.Model):
-    Product = models.ForeignKey(Product, 
-                                on_delete=models.CASCADE, 
-                                related_name='product_variations')
```

Comment:
If we have removed `primary_variant` then why we have still kept the Product Variation a bit unclear to me ?

Reply:
I just kept it as a stale model

Reply:
will clean up later